Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/git-touch/git-touch/) for GitTouch/GitTouch.



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/git-touch/-/git-touch/horizontal-auto.svg)

smooth80 commented:
merged

smooth80 reviewed (approved):
merge the language changes

Commit:
Translated using Weblate (Italian)

Currently translated at 98.8% (86 of 87 strings)

Translated using Weblate (German)

Currently translated at 100.0% (87 of 87 strings)

Translated using Weblate (French)

Currently translated at 100.0% (87 of 87 strings)

Translated using Weblate (French)

Currently translated at 100.0% (87 of 87 strings)

Added translation using Weblate (Italian)

Co-authored-by: J. Lavoie <j.lavoie@net-c.ca>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/de/
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/fr/
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/it/
Translation: GitTouch/GitTouch

RickyM7 committed:
chore: Translated using Weblate (Portuguese (Brazil))

Currently translated at 71.0% (118 of 166 strings)

Co-authored-by: Ricardo <contatorms7@tutamail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/pt_BR/
Translation: GitTouch/GitTouch

qwerty287 committed:
chore: Translated using Weblate (German)

Currently translated at 100.0% (166 of 166 strings)

chore: Translated using Weblate (German)

Currently translated at 66.2% (110 of 166 strings)

chore: Translated using Weblate (German)

Currently translated at 64.4% (107 of 166 strings)

chore: Translated using Weblate (German)

Currently translated at 57.8% (96 of 166 strings)

Co-authored-by: qwerty287 <ndev@web.de>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/de/
Translation: GitTouch/GitTouch

Vistaus committed:
chore: Translated using Weblate (Dutch)

Currently translated at 100.0% (166 of 166 strings)

chore: Translated using Weblate (Dutch)

Currently translated at 100.0% (166 of 166 strings)

Co-authored-by: Heimen Stoffels <vistausss@fastmail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/nl/
Translation: GitTouch/GitTouch

hikingc committed:
chore: Translated using Weblate (Spanish)

Currently translated at 57.8% (96 of 166 strings)

Co-authored-by: hiking <joaquinfc@protonmail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/es/
Translation: GitTouch/GitTouch

rezaalmanda committed:
chore: Translated using Weblate (Indonesian)

Currently translated at 99.3% (165 of 166 strings)

chore: Translated using Weblate (Indonesian)

Currently translated at 63.8% (106 of 166 strings)

Co-authored-by: Reza Almanda <rezaalmanda27@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/id/
Translation: GitTouch/GitTouch

Commit:
chore: Translated using Weblate (Czech)

Currently translated at 15.0% (25 of 166 strings)

chore: Added translation using Weblate (Czech)

Co-authored-by: Ondřej Sedláček <behests_phocaena@aleeas.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/cs/
Translation: GitTouch/GitTouch

ecxod committed:
chore: Translated using Weblate (Romanian)

Currently translated at 71.6% (119 of 166 strings)

chore: Translated using Weblate (German)

Currently translated at 100.0% (166 of 166 strings)

chore: Added translation using Weblate (Romanian)

Co-authored-by: Christian Eichert <c@zp1.net>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/de/
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/ro/
Translation: GitTouch/GitTouch

akyunus committed:
chore: Translated using Weblate (Turkish)

Currently translated at 83.1% (138 of 166 strings)

chore: Added translation using Weblate (Turkish)

Co-authored-by: Yunus AK <akyunus@live.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/tr/
Translation: GitTouch/GitTouch

Commit:
chore: Translated using Weblate (Chinese (Simplified))

Currently translated at 45.7% (76 of 166 strings)

Co-authored-by: Zhou Nan <zhounan@nfschina.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/zh_Hans/
Translation: GitTouch/GitTouch

simonaiacob committed:
chore: Translated using Weblate (Romanian)

Currently translated at 100.0% (166 of 166 strings)

chore: Translated using Weblate (Romanian)

Currently translated at 100.0% (166 of 166 strings)

chore: Translated using Weblate (Romanian)

Currently translated at 100.0% (166 of 166 strings)

Co-authored-by: Simona Iacob <s@zp1.net>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/ro/
Translation: GitTouch/GitTouch

Commit:
chore: Translated using Weblate (Chinese (Simplified))

Currently translated at 65.6% (109 of 166 strings)

Co-authored-by: Jane Kong <myonjane@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/zh_Hans/
Translation: GitTouch/GitTouch

SantosSi committed:
chore: Translated using Weblate (Portuguese)

Currently translated at 69.2% (115 of 166 strings)

chore: Translated using Weblate (Portuguese)

Currently translated at 71.0% (118 of 166 strings)

Co-authored-by: ssantos <ssantos@web.de>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/pt/
Translation: GitTouch/GitTouch

Lisapple committed:
chore: Translated using Weblate (French)

Currently translated at 95.1% (158 of 166 strings)

chore: Translated using Weblate (French)

Currently translated at 94.5% (157 of 166 strings)

chore: Translated using Weblate (French)

Currently translated at 93.9% (156 of 166 strings)

chore: Translated using Weblate (French)

Currently translated at 86.1% (143 of 166 strings)

chore: Translated using Weblate (French)

Currently translated at 50.0% (83 of 166 strings)

chore: Translated using Weblate (French)

Currently translated at 48.1% (80 of 166 strings)

Co-authored-by: Maxime Leroy <lisacintosh@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/fr/
Translation: GitTouch/GitTouch

nattadasu committed:
chore: Translated using Weblate (Indonesian)

Currently translated at 100.0% (166 of 166 strings)

Co-authored-by: Sultan Iskandar Maulana <nattadasu@nattadasu.my.id>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/id/
Translation: GitTouch/GitTouch

calcitem committed:
chore: Translated using Weblate (Chinese (Simplified))

Currently translated at 65.6% (109 of 166 strings)

Co-authored-by: Calcitem <calcitem@outlook.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/zh_Hans/
Translation: GitTouch/GitTouch

Commit:
chore: Translated using Weblate (Indonesian)

Currently translated at 99.3% (165 of 166 strings)

chore: Translated using Weblate (Indonesian)

Currently translated at 100.0% (166 of 166 strings)

chore: Translated using Weblate (Indonesian)

Currently translated at 100.0% (166 of 166 strings)

chore: Translated using Weblate (Indonesian)

Currently translated at 100.0% (166 of 166 strings)

Co-authored-by: Taufik <taufikadi.wicaksono@tutamail.com>
Co-authored-by: Taufik A. Wicaksono <taufikadi.wicaksono@tutamail.com>
Co-authored-by: Taufik Adi Wicaksono <taufikadi.wicaksono@tutamail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/id/
Translation: GitTouch/GitTouch

yangyangdaji committed:
chore: Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (166 of 166 strings)

chore: Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (166 of 166 strings)

Co-authored-by: yangyangdaji <1504305527@qq.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/zh_Hans/
Translation: GitTouch/GitTouch

fitojb committed:
chore: Translated using Weblate (Spanish)

Currently translated at 70.4% (117 of 166 strings)

Co-authored-by: Adolfo Jayme Barrientos <fitojb@ubuntu.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/es/
Translation: GitTouch/GitTouch

morotesovtannu committed:
chore: Translated using Weblate (Japanese)

Currently translated at 50.6% (84 of 166 strings)

Co-authored-by: Kon Kon Tannu Morote <kon2etonertannu@ilnk.info>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/ja/
Translation: GitTouch/GitTouch

HinameChan committed:
chore: Translated using Weblate (Turkish)

Currently translated at 84.3% (140 of 166 strings)

chore: Translated using Weblate (Turkish)

Currently translated at 83.7% (139 of 166 strings)

Co-authored-by: Kyoya <thelol9181@gmail.com>
Co-authored-by: kyoya <thelol9181@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/tr/
Translation: GitTouch/GitTouch

Srkrishna committed:
chore: Added translation using Weblate (Tamil)

Co-authored-by: srkrishna <srkrishna412@gmail.com>

Commit:
chore: Translated using Weblate (Spanish)

Currently translated at 100.0% (166 of 166 strings)

chore: Translated using Weblate (Spanish)

Currently translated at 100.0% (163 of 163 strings)

Co-authored-by: gallegonovato <fran-carro@hotmail.es>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/es/
Translation: GitTouch/GitTouch

antonlnz committed:
chore: Translated using Weblate (Galician)

Currently translated at 39.1% (65 of 166 strings)

chore: Added translation using Weblate (Galician)

Co-authored-by: Antón López <anton.lopez.nunez@udc.es>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/gl/
Translation: GitTouch/GitTouch

Commit:
chore: Translated using Weblate (Ukrainian)

Currently translated at 60.2% (100 of 166 strings)

chore: Added translation using Weblate (Ukrainian)

Co-authored-by: CakesTwix <cakestwix1@gmail.com>
Co-authored-by: Kiryazov Oleg <cakestwix1@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/uk/
Translation: GitTouch/GitTouch

Commit:
chore: Translated using Weblate (Indonesian)

Currently translated at 100.0% (166 of 166 strings)

Co-authored-by: EngageIndo <engageindo@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/id/
Translation: GitTouch/GitTouch

kchsoft committed:
chore: Added translation using Weblate (Korean)

Co-authored-by: Chang Hyeon Kim <kchranger@gmail.com>

Kefir2105 committed:
chore: Translated using Weblate (Ukrainian)

Currently translated at 100.0% (166 of 166 strings)

chore: Translated using Weblate (Ukrainian)

Currently translated at 61.4% (102 of 166 strings)

Co-authored-by: Dan <jonweblin2205@protonmail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/uk/
Translation: GitTouch/GitTouch

victorBuzdugan committed:
chore: Translated using Weblate (Romanian)

Currently translated at 100.0% (166 of 166 strings)

Co-authored-by: Victor Buzdugan <buzdugan.victor@icloud.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/ro/
Translation: GitTouch/GitTouch

NicoaraAlex committed:
chore: Translated using Weblate (Romanian)

Currently translated at 100.0% (166 of 166 strings)

chore: Translated using Weblate (Romanian)

Currently translated at 100.0% (166 of 166 strings)

Co-authored-by: Nicoara Alex <alex.nicoara@yahoo.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/ro/
Translation: GitTouch/GitTouch

Young-TW committed:
chore: Translated using Weblate (Chinese (Traditional))

Currently translated at 54.2% (90 of 166 strings)

Co-authored-by: Young <young20050727@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/zh_Hant/
Translation: GitTouch/GitTouch

Atalanttore committed:
chore: Translated using Weblate (German)

Currently translated at 98.7% (164 of 166 strings)

Co-authored-by: Ettore Atalan <atalanttore@googlemail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/de/
Translation: GitTouch/GitTouch

Commit:
chore: Translated using Weblate (Tamil)

Currently translated at 100.0% (166 of 166 strings)

chore: Translated using Weblate (Tamil)

Currently translated at 100.0% (166 of 166 strings)

chore: Translated using Weblate (Tamil)

Currently translated at 100.0% (166 of 166 strings)

Co-authored-by: தமிழ்நேரம் <anishprabu.t@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/ta/
Translation: GitTouch/GitTouch

Commit:
chore: Translated using Weblate (Russian)

Currently translated at 98.7% (164 of 166 strings)

Co-authored-by: gfbdrgng <hnaofegnp@hldrive.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/ru/
Translation: GitTouch/GitTouch

yurtpage committed:
chore: Translated using Weblate (Russian)

Currently translated at 100.0% (166 of 166 strings)

Co-authored-by: Yurt Page <yurtpage@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/ru/
Translation: GitTouch/GitTouch

jakob-info committed:
chore: Translated using Weblate (German)

Currently translated at 100.0% (166 of 166 strings)

Co-authored-by: Jakob <git@jakob-info.de>
Translate-URL: https://hosted.weblate.org/projects/git-touch/git-touch/de/
Translation: GitTouch/GitTouch
